### PR TITLE
feat(scheduler): implement Interval and IntervalAction Client

### LINKED ISSUE #572: [V2] Create support-scheduler client library
Create the client for interval and interval action APIs

### PR DESCRIPTION
**Pull request by weichou1229**
Close #572

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/go-mod-core-contracts/blob/master/.github/Contributing.md.

## What is the new behavior?
Add V2 Client:

- POST/PATCH /interval
- GET/DELETE /interval/name/{name}
- GET /interval/all

- POST/PATCH /intervalAction
- GET/DELETE /intervalAction/name/{name}
- GET /intervalAction/all

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## New Imports

- [ ] Yes
- [x] No

## Specific Instructions
Are there any specific instructions or things that should be known prior to reviewing?

## Other information

### CODE REVIEW

File: v2/clients/http/interval.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+}
+
+// NewIntervalClient creates an instance of IntervalClient
+func NewIntervalClient(baseUrl string) interfaces.IntervalClient {
+	return &IntervalClient{
+		baseUrl: baseUrl,
+	}
+}
+
+func (client IntervalClient) Add(ctx context.Context, reqs []requests.AddIntervalRequest) (
```
Comment (lenny-intel) [inline v2/clients/http/interval.go:34]: Add GoDoc comments for public functions in this PR.

**Review by lenny-intel — CHANGES_REQUESTED** [review]

File: v2/clients/http/interval.go (hunk 1 above)
Comment (weichou1229) [reply v2/clients/http/interval.go:35]: Added GoDoc comments.

**Review by lenny-intel — DISMISSED** [review]
LGTM

**Comment by weichou1229** [comment]
Rebased.

**Review by cloudxxx8 — APPROVED** [review]
LGTM

**Pull request merged**